pennylane_lightning/core/src/lightning_qubit.toml

```diff
@@ -0,0 +1,78 @@
+schema = 1
+
+[device]
+name = "lightning.qubit"
+precision = ['float32', 'float64']
+
+[operations]
+observables = [
+	"NamedObs",
+	"HermitianObs",
+	"TensorObs",
+	"HamiltonianObs",
+]
+
+[[operations.gates]]
+full = [
+	"Identity",
+	"PauliX",
+	"PauliY",
+	"PauliZ",
+	"Hadamard",
+	"S",
+	"T",
+	"PhaseShift",
+	"RX",
+	"RY",
+	"RZ",
+	"Rot",
+	"CNOT",
+	"CY",
+	"CZ",
+	"SWAP",
+	"IsingXX",
+	"IsingXY",
+	"IsingYY",
+	"IsingZZ",
+	"ControlledPhaseShift",
+	"CRX",
+	"CRY",
+	"CRZ",
+	"CRot",
+	"CSWAP",
+	"Toffoli",
+	"MultiRZ",
+        "SingleExcitation",
+        "SingleExcitationMinus",
+        "SingleExcitationPlus",
+        "DoubleExcitation",
+        "DoubleExcitationMinus",
+        "DoubleExcitationPlus",
+]
+# QubitUnitary should never be here.
+# Gates in "matrix" are decomposed to QubitUnitary.
+# Having a QubitUnitary here would mean an infinite loop.
+matrix = [
+	"OrbitalRotation",
+	"MultiControlledX",
+	# TODO:
+	# Why is qml.ops.Controlled not in qml namespace?
+	# TODO:
+	# other gates supported via applying matrices
+]
+
+[measurements]
+exactshots = [
+	"Expval",
+	"Var",
+]
+finiteshots = [
+	"Probs",
+	"Sample",
+	"Measure", # mid-circuit measurement
+]
```

Comment:
Are these intended to be disjoint sets or can there be overlap? Probs for example definitely supports exact results so could appear in both.

I believe the finite-shots capabilities are also being updated right now. The resulting table should then be:
```
exactshots = [
    "Expval",
    "Var",
    "Probs",
    "State",
]
finiteshots = [
    "Exp",
    "Var",
    "Probs",
    "Sample",
    "Counts",
]
```
Here I added `State` because I noticed it's missing.
There is also the question of whether `Measure` should appear here, as it is treated more like an operation than a measurement process. Measurement processes are always terminal in PennyLane, because they can imply repeated execution of the function they are defined within.

I think it could also make sense for there to be a separate Boolean property to explicitly declare support for mid circuit measurements or not.

Reply:
> measure will be in both operations and finite shots (no discussion about exact shots)

I think I would omit it from finite-shots, and restrict this list to measurement processes only.

Reply:
Removed and kept the boolean flag at the end.